index.html

```diff
@@ -32,6 +43,22 @@ <h3 class="tr">Add a new book</h3>
         <button class="add-btn" id="add-book-btn">Add</button>
       </div>
     </div>
+    <section class="contact-section hidden" id="contact-section">
+      <h2 class="contact-section-heading">Contact Information</h2>
+      <p>
+        Do have any question or you want to say "Hello"? <br />
+        You can reach out to us!
+      </p>
+      <ul>
+        <span>Our email: jumahqueenter@gmail.com</span>
+        <span>Our email: barazatracy44@gmail.com</span>
+        <span>Our phone number: +254710976397</span>
+        <span>Our phone number: +254769212978</span>
+        <span>Our address: Nairobi, Kenya, 5577-00100</span>
+      </ul>
+    </section>
+    <footer class="footer">All Rights are reserved .</footer>
```

Comment:
As per the wireframe provided by the microverse, the footer is at the bottom of the page. But in your case, it sticks to its nearest element. Kindly fix the issue.
| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/219011043-8ddb7678-f0bb-44c7-a7e7-2a731ed25b5e.png) | ![image](https://user-images.githubusercontent.com/33268927/219011299-7812f96a-5fce-4d78-bb43-0bdff36397f0.png) |